ReactiveCocoa/Swift/Signal.swift

```diff
@@ -263,36 +263,43 @@ private func observeWithStates<T, U, E>(signalState: CombineLatestState<T>, othe
 	}, interrupted: onInterrupted)
 }
 
-/// Combines the latest value of the receiver with the latest value from
-/// the given signal.
-///
-/// The returned signal will not send a value until both inputs have sent
-/// at least one value each. If either signal is interrupted, the returned signal
-/// will also be interrupted.
-public func combineLatestWith<T, U, E>(otherSignal: Signal<U, E>)(signal: Signal<T, E>) -> Signal<(T, U), E> {
+private func combineLatestWith<T, U, V, E>(otherSignal: Signal<U, E>, transform: ((T, U) -> V))(signal: Signal<T, E>) -> Signal<V, E> {
```

Comment:
This is no different than (the previous) `combineLatestWith` plus `map`. Can you please revert to that?